content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+
+The researcher reports that the average approval rate is between 2 and 3, meaning that users are slightly dissatisfied. Considering that the mean is 2.45, this is technically correct. But it is a disingenuous finding &mdash; only 25% of the users actually rated the product between a 2 and a 3.  
+
+It would be more accurate to report that the majority of users are either extremely dissatisfied or extremely satisfied with the product. By reporting the finding that the approval rate is between 2 and 3, the team might miss the root of the issue that causes disparities in the satisfaction of certain groups of users. </details> 
+
+### Interview Bias 
+
+__Interview bias__ refers to a participant's perception of the interviewer. This may be due to a user's own biased perception of an interviewer's ethnicity, age, gender, or physical appearance &mdash; or even details like body language or facial expressions. Interview bias can be difficult to avoid, so it is just important to keep in mind when evaluating the data. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+Imagine that a team of four researchers conducts user interviews. Three of the researchers are 25 years old and one researcher is 50 years old. All interview participants are 20-25 years old.  
+
+When reviewing the data, the team noticed a clear trend: participants who talked to the younger researchers shared detailed responses that indicated honesty and openness. By contrast, most participants who spoke with the older researcher shared little detail, and their responses indicated an unwillingness to share openly. In this case, participants may have held a bias against the older researcher, which impacted how they approached the interview. </details> 
+
+### Response Bias 
+
+__Response bias__ is the tendency to consciously or subconsciously provide inaccurate information due to the nature of self-reporting. If the research study relies on self-reported data, it's important to keep in mind the limitations that come with it. For reasons that don't pertain to the design, the user response may be inaccurate or misleading. A user may misremember information, leave out details that would be important, or change their responses to portray a certain self-image.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+If a survey asks, “How much time do you typically spend on a popular social media app every day?”, participants may not know the exact amount. Their guess could be inaccurate, or they may report a lower number if they feel guilty about spending too much time on social media. To minimize response bias in this scenario, researchers need a way to obtain more accurate data. Some phones automatically track how much time is spent on each app. Asking participants to reference the data tracked by their phone should lead to more accurate responses. </details> 
```

Comment:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

If a survey asks, “How much time do you typically spend on a popular social media app every day?”, participants may not know the exact amount. Their guess could be inaccurate, or they may report a lower number if they feel guilty about spending too much time on social media. To minimize response bias in this scenario, researchers need a way to obtain more accurate data. Some phones automatically track how much time is spent on each app. Asking participants to reference the data tracked by their phone should lead to more accurate responses. </details> 
```
```